client/src/plus/collections/collection.tsx

```diff
@@ -110,10 +111,72 @@ export default function CollectionComponent() {
   );
 }
 
+function FrequentlyViewedCollectionComponent() {
+  let [size, setSize] = useState(0);
+  let [atEnd, setAtEnd] = useState(false);
+  let frequentlyViewed = useFrequentlyViewed(10, size, setAtEnd);
+  let _vals: Item[] | undefined = frequentlyViewed?.items.map((v) => {
+    return {
+      collection_id: frequentlyViewed?.name || "",
```

Comment:
This makes me a little uncomfortable - this isn't an actual collection id, querying the collections api with it will return errors. Generally, forcing the FV data into the exact shape of a collection `Item` feels like it could lead to bugs in the future.

I'd perhaps add a new `FrequentlyViewedItem` type, with most of the same properties as the `Item` type (less `collection_id`, and I guess `notes: undefined` as we can't conditionally call the `item.notes` `useEffect` hook) and make the `ItemComponent` take either.